scout/server/blueprints/cases/controllers.py

```diff
@@ -594,7 +594,7 @@ def _get_decorated_var(var_obj: dict) -> dict:
         case_id=case_obj["_id"], institute_id=institute_obj["_id"]
     ):
         for eval_category, variant_key in CASE_REPORT_VARIANT_TYPES.items():
-            if var_obj.get(variant_key):
```

Comment:
That's because bool(0) is False?

Reply:
Yes..

Reply:
Ah, if I think about it, `None` values may be there as well. Hm, that should have been handled downstream, but maybe let me add an explicit None-check as well.